templates/terraform/post_create/set_computed_name.erb

```diff
@@ -1,7 +1,11 @@
 // `name` is autogenerated from the api so needs to be set post-create
 name, ok := res["name"]
 if !ok {
-  return fmt.Errorf("Create response didn't contain critical fields. Create may not have succeeded.")
+	name, ok = res["response"].(map[string]interface{})["name"]
```

Comment:
If an api response doesn't have "name" or "response" for whatever reason, wouldn't this be a nil ptr?